Marlin/src/HAL/HAL_STM32/HAL.h

```diff
@@ -54,7 +54,7 @@
   #error "SERIAL_PORT must be from -1 to 6"
 #endif
 #if SERIAL_PORT == -1
-  #define MYSERIAL0 SerialUSB
+  #define MYSERIAL0 Serial
```

Comment:
This is fine if everyone is selecting USB CDC superceding Serial, but that isn't always true, is it?